diff --git a/node_modules/react-gantt-timeline/src/lib/components/common/ContentEditable.js b/node_modules/react-gantt-timeline/src/lib/components/common/ContentEditable.js
index 8b6d5ca..4be330b 100644
--- a/node_modules/react-gantt-timeline/src/lib/components/common/ContentEditable.js
+++ b/node_modules/react-gantt-timeline/src/lib/components/common/ContentEditable.js
@@ -63,7 +63,8 @@ export default class ContentEditable extends Component{
                     name="name" 
                     value={this.state.value} 
                     onKeyUp={this.handleKey}
-                    onChange={this.handleChange}/>
+                    onChange={this.handleChange}
+                    readonly/>
     }
 
     render(){
diff --git a/node_modules/react-gantt-timeline/src/lib/components/viewport/DataTask.js b/node_modules/react-gantt-timeline/src/lib/components/viewport/DataTask.js
index f7794c2..9b02a07 100644
--- a/node_modules/react-gantt-timeline/src/lib/components/viewport/DataTask.js
+++ b/node_modules/react-gantt-timeline/src/lib/components/viewport/DataTask.js
@@ -116,8 +116,8 @@ export default class DataTask extends Component{
                 <div  className="timeLine-main-data-task-side-linker" 
                     onMouseUp={(e)=>this.onCreateLinkMouseUp(e,LINK_POS_LEFT)}/>
             </div>
-            <div style={{overflow:'hidden'}}>
-            {Config.values.dataViewPort.task.showLabel?this.props.item.name:""}
+            <div className="hola" style={{overflow:'hidden'}}>
+            {Config.values.dataViewPort.task.showLabel?this.props.item.description:""}
             </div>
             <div className="timeLine-main-data-task-side" 
                  style={{top:0,left:style.width-3,height:style.height}}
